public/app/features/manage-dashboards/components/PublicDashboardListTable.tsx

```diff
@@ -0,0 +1,95 @@
+import { css } from '@emotion/css';
+import React, { useState } from 'react';
+import useAsync from 'react-use/lib/useAsync';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { Link, ButtonGroup, LinkButton, Icon, Tag, useStyles2 } from '@grafana/ui';
+import { getConfig } from 'app/core/config';
+
+export interface ListPublicDashboardResponse {
+  uid: string;
+  accessToken: string;
+  dashboardUid: string;
+  title: string;
+  isEnabled: boolean;
+}
+
+export const LIST_PUBLIC_DASHBOARD_URL = `/api/dashboards/public`;
+export const getPublicDashboards = async () => {
+  const resp: ListPublicDashboardResponse[] = await getBackendSrv().get(LIST_PUBLIC_DASHBOARD_URL);
+  return resp.sort((a, b) => Number(b.isEnabled) - Number(a.isEnabled));
+};
+
+export const viewPublicDashboardUrl = (accessToken: string): string => {
+  return `${getConfig().appUrl}public-dashboards/${accessToken}`;
+};
+
+export const ListPublicDashboardTable = () => {
+  const styles = useStyles2(getStyles);
+  const [publicDashboards, setPublicDashboards] = useState<ListPublicDashboardResponse[]>([]);
+
+  useAsync(async () => {
+    const publicDashboards = await getPublicDashboards();
+    setPublicDashboards(publicDashboards);
+  }, [setPublicDashboards]);
+
+  return (
+    <div className="page-action-bar">
+      <table className="filter-table">
+        <thead>
+          <tr>
+            <th>Dashboard</th>
+            <th>Public dashboard Enabled</th>
```

Comment:
We use sentence case so this should be changed to `Public dashboard enabled`. Also maybe a title like `status` or just `enabled` is more accurate since we are already in the context of public dashboards.